src/Libraries/Microsoft.Extensions.AI/ChatCompletion/FunctionInvocationContext.cs

```diff
@@ -94,4 +94,10 @@ public IList<ChatMessage> Messages
     /// more function call requests in responses.
```

Comment:
We should update this to also mention the sequential behavior, e.g. adding to the end
```
If multiple function call requests are issued as part of a single iteration (a single response from the inner <see cref="IChatClient"/>), setting  <see cref="Terminate" /> to <see langword="true" /> may also prevent subsequent requests within that same iteration from being processed.
```